addons/mail/static/src/js/activity.js

```diff
@@ -333,6 +333,8 @@ var BasicActivity = AbstractField.extend({
                 $popover.find('#activity_feedback').focus();
                 self._bindPopoverFocusout($(this));
             }).popover('show');
+        } else {
+            $markDoneBtn.data('bs.popover').show();
```

Comment:
flicker when clicking twice on `textarea`.
Something like this should fix this issue:

```js
var popover = $markDoneBtn.data('bs.popover');
if ($('#' + popover.tip.id).length === 0) {
   popover.show();
}
```